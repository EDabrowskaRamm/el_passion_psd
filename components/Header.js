import Link from 'next/link'

export default () =>
  <div>
    <Link href='/'><a>home</a></Link>
    <br/>
    <Link href='/about'><a>about</a></Link>
  </div>
