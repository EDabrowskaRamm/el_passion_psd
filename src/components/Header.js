import Link from '~/components/Link'
import Logo from '~/svg/logo.svg'

export default () =>
  <div className='container'>
    <Link href='/'>
      <Logo width='180px' />
    </Link>
    <Link href='/about'>about</Link>
  </div>
