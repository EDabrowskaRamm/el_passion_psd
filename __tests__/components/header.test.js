/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Header from '~/components/Header.js'

describe('Header', () => {
  it('Snapshot', () => {
    const component = renderer.create(<Header />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
