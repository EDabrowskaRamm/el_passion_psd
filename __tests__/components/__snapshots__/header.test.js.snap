// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot 1`] = `
<header>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      src={
        Object {
          "process": [Function],
        }
      }
      srcSet=""
      width="300"
    />
  </a>
</header>
`;
